docs/diginstroom/sip/2.0/profiles/film.md

```diff
@@ -0,0 +1,251 @@
+    │   │   └── mezzanine.mov                           # Playable mezzanine
+    │   └── metadata
+    │       └── preservation
+    │           └── premis.xml
+    │
+    └── representation_3
+        ├── METS.xml
+        ├── data
+        │   └── QC.dpx                                  # DPX for QC
+        └── metadata
+            └── preservation
+                └── premis.xml
+```
+
+## Requirements
+
+### General
+
+- A SIP MUST contain content of exactly one digitised film, consisting of one or more image and/or audio reels.
+- <mark> The content MUST be digitised per reel, i.e. each ZIP and/or MOV and/or DPX file contained in their respective representation directories MUST represent exactly one film consisting of one or more image and/or audio reels.</mark>
+- There MUST be exactly one IE present in the SIP, i.e. the digitised film.
+- There MUST be preservation metadata at the package level in the `preservation/premis.xml` file.
+- There MUST be preservation metadata at the representation level in the respective `preservation/premis.xml` files.
+- Preservation metadata in the SIP MUST be limited to the PREMIS metadata schema.
+- Fixity MUST be calculated using the MD5 hashing algorithm, thus:
+  - The value of element `premis:premis/premis:object[@xsi:type="premis:file"]/premis:objectCharacteristics/premis:fixity/premis:messageDigestAlgorithm` MUST be set to `MD5`.
+  - The value of attribute `premis:premis/premis:object[@xsi:type="premis:file"]/premis:objectCharacteristics/premis:fixity/premis:messageDigestAlgorithm/@valueURI` MUST be set to `"http://id.loc.gov/vocabulary/preservation/cryptographicHashFunctions/md5"`.
+  - The value of all `//*/@CHECKSUMTYPE` attributes in the `METS.xml` files MUST be set to `MD5`.
+- <mark>There MAY be descriptive metadata at the representation level (e.g. information about the representations, such as a title or a description).</mark>
+
+### Package METS
+
+- The `/mets/@TYPE` attribute MUST be set to `Video – File-based and Physical Media`.
+- The `csip:CONTENTINFORMATIONTYPE` attribute MUST be set to `OTHER` and the `csip:OTHERCONTENTINFORMATIONTYPE` attribute MUST be set to `https://data.hetarchief.be/id/sip/2.0/film`.
+- The `mets/dmdSec/mdRef/@MDTYPE` attribute MUST be set to `OTHER` and the `mets/dmdSec/mdRef/@OTHERMDTYPE` MUST be set to `DC+SCHEMA`.
+
+### Package Descriptive Metadata
+
+- A descriptive metadata file `descriptive/dc+schema.xml` describing the IE  MUST be present at the package level.
+- Descriptive metadata in the `descriptive/dc+schema.xml` MUST be limited to the DCTERMS and SCHEMA elements outlined in the [basic profile](https://developer.meemoo.be/docs/diginstroom/sip/2.0/profiles/basic.html#dc-requirements).
+- The DCTERMS and SCHEMA metadata in the `descriptive/dc+schema.xml` file MUST follow the [basic profile requirements](https://developer.meemoo.be/docs/diginstroom/sip/2.0/profiles/basic.html#dc-requirements) regarding the use of elements and attributes.
```

Comment:
The first line in that link: 
> There MUST NOT be any descriptive metadata at the representation level.

Which isn't the case for this profile.

Reply:
I eliminated any  mention of descriptive metadata at the representation level since I currently don't know of/see any use case. So this contradiction should be resolved.

In any case, the link that is used has mostly to do with the elements that are outlined in that section and not with the requirements stated in it.